p5js/Crepe_Game/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="UTF-8">
+
+  <title>Pitch Tone</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
+  <script src="./p5.sound.js"></script>
+  <script language="javascript" type="text/javascript" src="../../dist/ml5.js"></script>
```

Comment:
Oh, this too! i.e. `<script src="https://unpkg.com/ml5@_._._/dist/ml5.js"></script>`